css/colorGame.css

```diff
@@ -0,0 +1,75 @@
+body{
+	background: #232323;
+	margin: 0;
+	text-transform: uppercase;
+	/*font-family: 'Poppins', sans-serif;*/
+}
+
+.square{
```

Comment:
Please, how come you're pushing a javacript code into this branch ????
Is this meant to be? It's causing confusions for others! We shouldn't be cloning your codes or whatever!
Please find a way to remove this!